protocol/avdl/keybase1/teams.avdl

```diff
@@ -757,6 +769,11 @@ protocol teams {
     SignatureMetadata sigMeta;
   }
 
+  record TeamUsedInviteLog {
+    UserVersion uv;
+    int logPoint; // index into UserLogPoint array for UV
```

Comment:
not opposed but is this for something in particular besides tests?

Reply:
in case there is a need for a list of who and when joined using that invite

Reply:
Update the comment to `TeamSigChainState.userLog[uv]` or similar